addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -349,7 +349,9 @@ Composer.defaultProps = {
     hasDiscardButton: false,
     hasFollowers: false,
     hasSendButton: true,
+    hasTextInputSendOnEnterEnabled: true,
     hasThreadName: false,
+    isCompact: true,
```

Comment:
I don't really get what it means "compact" for this composer.
Intuitively, a component should not be "compact" by default.
This is especially intriguing when all added SCSS are `:not(isCompact)`

Reply:
Compact is the way it is used and printed in chat windows and discuss, not compact is the way it's printed in chatter. I didn't find a better name than this one ..

Reply:
So, do you have a suggestion ?